src/App.js

```diff
@@ -1,14 +1,39 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import Todo from './components/TodoComponents/Todo';
+
+
+
+let todoItem = [{task : '#1', id : Date.now(),  completed : false}];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(props){
+    super(props);
+    this.state = { items: todoItem};
+    this.addTodo = this.addTodo.bind(this);
+  }
+
+  addTodo(item) {
+    let  newItem = {task: item, id: Date.now(), completed: false };
```

Comment:
this isn't quite right, you need to get the value for task from `this.state`, and to do that, you need to get the input form to be updating state with the string you're entering